java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestItemUtils.java

```diff
@@ -26,70 +25,53 @@
 import org.eclipse.jdt.ls.core.internal.JDTUtils;
 import org.eclipse.lsp4j.Range;
 
-import java.util.Optional;
+import static java.util.Collections.emptyList;
 
 @SuppressWarnings("restriction")
 public class TestItemUtils {
-    protected static final String DISPLAY_NAME_ANNOTATION_JUNIT5 = "org.junit.jupiter.api.DisplayName";
 
     public static TestItem constructTestItem(IJavaElement element, TestLevel level) throws JavaModelException {
         return constructTestItem(element, level, null);
     }
 
     public static TestItem constructTestItem(IJavaElement element, TestLevel level, TestKind kind)
             throws JavaModelException {
-        String displayName = element.getElementName();
-        if (kind == TestKind.JUnit5 && element instanceof IMethod) {
-            final Optional<IAnnotation> annotation = TestFrameworkUtils.getAnnotation((IMethod) element,
-                    DISPLAY_NAME_ANNOTATION_JUNIT5);
-            if (annotation.isPresent()) {
-                displayName = (String) annotation.get().getMemberValuePairs()[0].getValue();
-            }
-        }
+        final String displayName = element.getElementName();
+        final String fullName = parseTestItemFullName(element, level);
+        final String uri = JDTUtils.getFileURI(element.getResource());
+        final Range range = parseTestItemRange(element);
+        final String projectName = element.getJavaProject().getProject().getName();
 
-        return new TestItem(displayName, parseTestItemFullName(element, level),
-                JDTUtils.getFileURI(element.getResource()), parseTestItemRange(element, level), level, kind,
-                element.getJavaProject().getProject().getName());
+        return new TestItem(displayName, fullName, uri, range, level, kind, projectName, emptyList());
```

Comment:
How about `Collections.emptyList()` instead of static importing?  Had a glance at the [ref](https://stackoverflow.com/questions/420791/what-is-a-good-use-case-for-static-import-of-methods), it looks like people are avoiding using static importing unless some constants are heavily used.